bfe_basic/common.go

```diff
@@ -80,7 +80,7 @@ func CreateInternalResp(request *Request, code int) *bfe_http.Response {
 	return res
 }
 
-// this interface is used for lookup config for each request
+// ServerDataConfInterface this interface is used for lookup config for each request
```

Comment:
// ServerDataConfInterface is an interface used for lookup config for each request